mp/src/game/client/momentum/clientmode_mom_normal.cpp

```diff
@@ -193,6 +193,10 @@ int ClientModeMOMNormal::HudElementKeyInput(int down, ButtonCode_t keynum, const
                 m_pSpectatorGUI->SetMouseInputEnabled(true);
             return 0;
         }
+
+        static ConVarRef mom_mapfinished_movement_enable("mom_mapfinished_movement_enable");
```

Comment:
This convar is on the client project and can just be `extern`'d

Reply:
Oh, I'm getting an error saying it's an unresolved extern, unsure if I'm missing something or misunderstanding you?

Reply:
You may have to make it not static in the original file it's found in.